### ISSUE TITLE
Embed a Blazor component inside an Angular App needs very long to be rendered

### ISSUE THREAD

**Issue by truthz03** [dotnet/aspnetcore#58831]
Hi,

I tried this [example](https://github.com/javiercn/CustomElementsBlazorSample) because I need to render some Blazor components inside my Angular app.
I choose this example because I think it was an overkill to load the whole webassembly stuff and therefore I want to use serverside rendering.

The example is working but it took several minutes to show the counter component. In the meantime the page is completely empty.
After the component was rendered the first time and refreshing the browser tab it renders faster (about 1s)

What causes this reeeeaaaly long delay?

cc: @guardrex

**Comment by mkArtakMSFT** [comment]
@truthz03 were you by any chance experiencing this during a debug session? Can you try to run the app without the debugger?

**Comment by truthz03** [comment]
Hi, because my real app where I want to use this mechanism is Net8, I replaced net7.0 by net8.0 inside the project file and updated the Nuget packages.
Now it is working like a sharm.

**Comment by guardrex** [comment]
That doesn't make me feel great because it should be supported back to .NET 6. Given that your code is working, I guess we'll just take further feedback on the guidance.

**Comment by javiercn** [comment]
@guardrex I don't think this is an issue, especially since .NET 6.0 is going out of support soon.
